person.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'module_highfiveable'
+
+class Person
+  attr_accessor :name
+  attr_reader :age
+
```

Comment:
All people have a phase, might as well put the attr_reader here, even though you don't initialize it here.